views/js/OystOneClick.js

```diff
@@ -62,7 +61,7 @@ function OystOneClick(url, productId) {
             available = false;
         }
 
-        if (productExported.isExported && available) 
+        if (available) 
```

Comment:
That's not the way to know if the product is available or not, using the css that way is bad.